Lib/profile.py

```diff
@@ -25,10 +25,8 @@
 
 
 import sys
-import os
```

Comment:
def trace_dispatch (line 184 in 3.6.1) is preceded by an apparently out-of-date comment
    # Heavily optimized dispatch routine for os.times() timer
That os.times is no longer used is why the os import can be moved.
self.timer is now  time.process time